node_modules/oae-core/topnavigation/css/topnavigation.css

```diff
@@ -73,6 +79,10 @@
     position: relative;
 }
 
+#topnavigation-right .topnavigation-haveaccount {
```

Comment:
The "Already have an account" label is not vertically aligned with the Sign In link. It can also be moved 5 or 10px closer to the Sign In link.

<img width="288" alt="screen shot 2015-09-09 at 22 14 47" src="https://cloud.githubusercontent.com/assets/109850/9780648/4482397c-5740-11e5-97bd-083ad4d480ec.png">